Makefile

```diff
@@ -38,6 +38,7 @@ EXCLUDE_SRC_FROM_LIB+=$(foreach file, $(SRCDIR)/main,$(foreach cext,$(CEXTS),$(f
 TEMPLATE_FILES=$(ROOT)/common.mk $(FWDIR)/v5.ld $(FWDIR)/v5-common.ld $(FWDIR)/v5-hot.ld
 TEMPLATE_FILES+=$(FWDIR)/libc.a $(FWDIR)/libm.a
 TEMPLATE_FILES+= $(INCDIR)/api.h $(INCDIR)/main.h $(INCDIR)/pros/*.* $(INCDIR)/display
+TEMPLATE_FILES+= $(INCDIR)/rtos/FreeRTOS.h $(INCDIR)/rtos/FreeRTOSConfig.h $(INCDIR)/rtos/portable.h $(INCDIR)/rtos/portmacro.h $(INCDIR)/rtos/projdefs.h
```

Comment:
oh, i see what you meant now. not sure I like this

Reply:
will have to think on the solution, might be just pulling the relevant structures and types out into this file as well...

Reply:
Trying to. I think we will make it to 1800 lines.

Reply:
Updated